CREDITS.md

```diff
@@ -0,0 +1,27 @@
+## Contributors
+The Voting Information Project is a collaboration between Google and the Pew
+Charitable Trusts, with input from election officials and other members of the
+public. In order to meet the attribution requirements in the
+[license](LICENSE.md), please use "The Voting Information Project" to signify
+the source of the content. The specific list of contributors and their GitHub
+usernames are included below.
+
+### Google
+* Justin Moore (jdmgoogle)
+* Jesse Mwaura (jmwaura)
+* Jonathan Tomer (jktomer)
+
+### Pew (and contractors)
+* Jared Marcotte (jungshadow)
+* Paul Stenbjorn (pstenbjorn)
+* Noma Thayer (nomadaisy)
+* Jen Tolentino (jen-tolentino)
+* Paul Kominers (pkoms)
+* David McGivney (decmg & df-dave)
```

Comment:
Dave should move down to "other" (_**NB:** maybe we should rename "other" to "Independent Contributors" or similar_).